lib/theme/spacings.dart

```diff
@@ -0,0 +1,16 @@
+/**
+ * Spacing naming follow similar convention as Material Design colors,
+ * so that it's not necessary to have spacings like xxxxxsmall etc.
+ * Three main spacings are defined as `small`, `medium` and `large`
+ * and number next to it is the percentage value of the base size.
+ * For example `small_150` = `small_100` * 150%
+ */
+abstract class Spacing {
```

Comment:
@JElgar Another helper class to unify spacings, please let me know if that's not an overkill

Reply:
I think we already have something similar here: https://github.com/now-u/now-u-app/blob/main/lib/assets/constants.dart#L80. Potentially we should move that and the assets into that file?

Reply:
Ah I wasn't aware of that, thanks, I'll migrate it